packages/eslint-plugin-next-on-pages/src/rules/no-unsupported-configs.ts

```diff
@@ -225,9 +232,52 @@ function getExportedConfigName(ast: Program): string | null {
 			/*  empty */
 		}
 	}
+
+	// we get the node right after /** @type {import('next').NextConfig} */
+	const nodeAfterNextConfigComment = getNodeAfterComment(
```

Comment:
Could we flip this and use the eslint API for `getCommentsBefore(nodeOrToken)` on each node in the AST.
That would avoid having to write and maintain our own node after comment helper.

Reply:
ah wow, awesome, sorry I didn't think eslint provided something like this out of the box 😓

Reply:
I've refactored the code to use that 🙂👍